### PR TITLE
Fix/update theme styles 404 page

### LINKED ISSUE #700: Fix existing logo for 2.1
Design - https://app.zeplin.io/project/5bcd9657767a220982a4ab61/screen/5df3a3f39fb143b164ea71ba

- [ ] Replace current 2.0 logo with 2.1
- [ ] Ensure colors are correct for corporate theme.

### PR DESCRIPTION
**Pull request by raichev-dima**
## Description

Added the "Corporate" theme styles support.
Closes #700

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/iconset/VaIconVuestic.vue [hunk 1]
```diff
@@ -7,41 +7,51 @@
...
       />
     </g>
   </svg>
 </template>
 <script>
+import { themetId } from '../components/pages/404-pages/ThemeConfig'
```
Comment (asvae) [inline src/iconset/VaIconVuestic.vue]: We don't really need this enum. Keeping corporate and original version is just a hack and we want to limit it to one computed property ideally. 
Also, some comments on why we use inverted color flag for theme change would be welcome.

File: src/components/pages/404-pages/VaPageNotFound.vue [hunk 2]
```diff
@@ -65,24 +80,6 @@ export default {
...
-    }
-
-    &__wallpaper {
-      position: absolute;
-      bottom: 0;
-      left: 1rem;
-      width: 30%;
-      height: 40%;
-      background-color: transparent;
-      background-image: url("../../../assets/petro.svg");
```
Comment (asvae) [inline src/components/pages/404-pages/VaPageNotFound.vue]: This svg is never used, so we can delete the file.

File: src/components/pages/404-pages/Wallpaper.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<script>
+export default {
+  props: {
+    color: String,
+  },
+}
+</script>
+
+<style lang="scss">
+.page-not-found__wallpaper {
```
Comment (asvae) [inline src/components/pages/404-pages/Wallpaper.vue]: Might make sense to do positioning in parent and keep this svg being svg?
Up to you.

**Comment by ilearnjs** [comment]
Bug for wide screens
https://github.com/epicmaxco/vuestic-admin/issues/706

**Pull request merged**